recipes/nccl/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "nccl" %}
+{% set version = "2.4.6" %}
+{% set revision = "1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}.{{ revision }}
+
+source:
+  url: https://github.com/NVIDIA/nccl/archive/v{{ version }}-{{ revision }}.tar.gz
+  sha256: ea4421061a7b9c454f2e088f68bfdbbcefab80ce81cafc70ee6c7742b1439591
+  patches:
+    - opt_sel_cudart_lib.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  run_exports:
+    # xref: https://github.com/NVIDIA/nccl/issues/218
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - {{ compiler("cuda") }}
+    - make
+  host:
+  run:
```

Comment:
You might want to remove these lines to see if the recipe can lint

Reply:
It would be nice for the linter not to error on this. ( https://github.com/conda-forge/conda-smithy/issues/1153 ) 🙂